charts/osbuilder/templates/nginx-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+    name: osartifactbuilder-operator-nginx-public
+    namespace: '{{.Release.Namespace}}'
+spec:
+    accessModes:
+      - ReadWriteOnce
```

Comment:
ditto. it is pretty sensible depending on the backing storage driver, and some might want to use ReadWriteMany if used in other contexts

Reply:
yes, improvements for future PRs